src/Main.purs

```diff
@@ -2,18 +2,27 @@ module Main where
 
 import Prelude
 
+import Control.Monad.Except.Trans (ExceptT(..), runExceptT)
 import Data.Argonaut.Core (Json)
+import Data.Either (Either(..))
 import Data.Foldable (traverse_)
 import Effect (Effect)
-import Effect.Aff (launchAff_)
+import Effect.Aff (launchAff_, runAff_)
+import Effect.Class (liftEffect)
+import Effect.Exception (message)
+import GitHub.Actions.Core as Core
 import Setup.BuildPlan (constructBuildPlan)
 import Setup.GetTool (getTool)
 import Setup.UpdateVersions (updateVersions)
 
 main :: Json -> Effect Unit
-main json = do
-  tools <- constructBuildPlan json
-  launchAff_ $ traverse_ getTool tools
+main json = runAff_ go $ runExceptT do
+  tools <- ExceptT (liftEffect (runExceptT (constructBuildPlan json)))
```

Comment:
Is this mapExceptT liftEffect?

Reply:
Oh yeah! Knew I was forgetting something